impeller/renderer/backend/metal/surface_mtl.h

```diff
@@ -58,6 +58,10 @@ class SurfaceMTL final : public Surface {
   // Returns a Rect defining the area of the surface in device pixels
   IRect coverage() const;
 
+  void PresentWithTransaction(bool present_with_transaction) {
```

Comment:
Ditto nitpick: add a minimal doc comment. It's super obvious from the implementation, but for people flashing up an IDE tooltip it's nice to know this is just a setter not actually calling `Present` under the hood. Could also just be `SetPresentWithTransaction` but that reads a bit grosser :/